src/server/index.js

```diff
@@ -91,6 +98,11 @@ if (fs.existsSync(customConfigPath)) {
         ...customConfig.module.loaders || [],
       ],
     },
+    resolve: {
```

Comment:
Just realized the replacement could be achieved using `NormalModuleReplacementPlugin`.